alf/environments/torch_environment.py

```diff
@@ -141,50 +143,38 @@ def current_time_step(self):
         return self._current_time_step
 
     def reset(self):
-        """Starts a new sequence and returns the first `TimeStep` of this sequence.
+        """Starts a new sequence and returns the first ``TimeStep`` of this sequence.
 
         Note: Subclasses cannot override this directly. Subclasses implement
-        _reset() which will be called by this method. The output of _reset() will
-        be cached and made available through current_time_step().
+        ``_reset()`` which will be called by this method. The output of ``_reset()``
+        will be cached and made available through ``current_time_step()``.
 
         Returns:
-            A `TimeStep` namedtuple containing:
-                step_type: a `Tensor` of `StepType` enum values.
-                reward: a `Tensor` of reward values from executing 'prev_action'.
-                discount: A discount value in the range `[0, 1]`.
-                observation: A (nested) 'Tensor' for observation
-                prev_action: A (nested) 'Tensor' for action from previous time step
-                env_id: the ID of the environment from which this time_step is
+            TimeStep:
```

Comment:
What do you mean?

Reply:
The comment ends with ":".  Looks incomplete.  Is that for formatting?  Maybe I'll just have to get used to it..

Reply:
The correct format for "Returns" is `type: description`. When the type says everything we can just ignore the `description`, but ':' is still needed in order for sphinx to interpret the token before ':' is a type. Otherwise it's just plain text